frontend/src/components/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,10 @@
+import {Header} from "../Header.tsx";
+import {Footer} from "../Footer.tsx";
+
+export function ProfilePage() {
+    return <>
+        <Header/>
+        <p>profile page</p>
+        <Footer/>
```

Comment:
I would suggest using Header and Footer in App.tsx (= only use the component once). If there are pages where they should not be displayed, you can use a state boolean to show/hide them.